tests/test1010.py

```diff
@@ -0,0 +1,54 @@
+# test some libuninameslist functions (if the library is included).
+
+import sys, fontforge
+
+cnt = UnicodeBlockCountFromLib()
+if ( cnt < 1 ):
```

Comment:
This is python, not C, remove the brackets.

Reply:
Didn't want it to be confused as a variable :-)
...but it appears to be common:
https://stackoverflow.com/questions/19747371/python-exit-commands-why-so-many-and-when-should-each-be-used
...even here, there is an example where you need to say quit() exit(), not quit or exit.

Reply:
Functions have nothing to do with if statements. Having brackets where they are unnecessary in such statements is un-pythonic. Just remove them.